src/ui/Analytics/Analytics.ts

```diff
@@ -405,7 +424,7 @@ export class Analytics extends Component {
     if (this.disabled || this.client instanceof NoopAnalyticsClient) {
       return this.logger.warn('Could not clear local data while analytics are disabled.');
     }
-    this.client.endpoint.clearCookies();
```

Comment:
Related to the above comment, I think we can potentially avoid a breaking change by placing the `clearCookies` logic back in the endpoint class. What was the reason for moving the logic up here?

Reply:
I wanted to read, write and clear cookies in the same location. However, this was otherwise unnecessary and I'm reverting it.